src/components/Inputs/TextInput/TextInputWithTopPlaceHolder.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import TextInput, { TextInputProps } from './TextInput';
+import styles from './TextInput.module.scss';
+
+interface TextInputWithTopPlaceHolderProps extends TextInputProps {
+	topPlaceholder: string;
+}
+
+const TextInputWithTopPlaceHolder: React.FC<TextInputWithTopPlaceHolderProps> =
```

Comment:
I think this should be a feature of TextInput itself instead of a separate component. We can handle this later though.

Reply:
Shall add to tech debt notes